lib/manageiq/providers/workflows/runner.rb

```diff
@@ -47,23 +47,21 @@ def delete_workflow_instance(workflow_instance)
         attr_accessor :docker_wait_thread
 
         def docker_wait
-          loop do
```

Comment:
I moved the loop up a level so it was easier to test this method in isolation, think it actually makes more sense since the loop is just for the thread to continue if the wait terminates prematurely for some reason.

Reply:
What breaks out of the loop though?